third_party/com_github_aws_c_common/system.BUILD.bazel

```diff
@@ -0,0 +1,4 @@
+cc_library(
```

Comment:
We probably don't need the system.BUILD.bazel files here; since it's c++, and since there are not common packages, we will likely build from source for now.  Though maybe at some point conda-forge will want this.

https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/sdk-from-pm.html